test/parallel/test-crypto-pbkdf2.js

```diff
@@ -111,8 +111,8 @@ assert.throws(
     }, {
       code: 'ERR_OUT_OF_RANGE',
       name: 'RangeError',
-      message: 'The value of "keylen" is out of range. It ' +
-               `must be >= 0 && < 4294967296. Received ${input}`
+      message: 'The value of "keylen" is out of range. It must be ' +
+               `>= 0 && < 4294967296. Received ${input.toLocaleString()}`
```

Comment:
This needs to be changed since `toLocaleString()` is no longer used?

Reply:
Thanks. Fixed.